Source/MonoGame.Extended/Animations/Easing/CurveEasing.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.Xna.Framework;
+
+namespace MonoGame.Extended.Animations.Easing
+{
+    public class CurveEasing : EasingFunction
+    {
+        public CurveEasing(Curve curve) {
```

Comment:
Isn't this possible using source control somehow? I'm not too familiar but I've heard of automatic bracket styling somewhere before.